### DISCUSSION TITLE
Trouble displaying tapped on images from README files

### DISCUSSION THREAD

**Discussion by marlonjames71** [community/community#48398] [Mobile]
**Repo**
https://github.com/aheze/Setting

**Reproduce**
On the iOS app, tap on any of the images in the README. Instead of showing the image, there's a screen with a "Error loading image" message and a button: "Try Again".

![image](https://user-images.githubusercontent.com/48145861/220945598-3360b44c-87e2-4e1e-a133-a3c2bfc0a4e3.png)
![image](https://user-images.githubusercontent.com/48145861/220945635-8f26e4b2-37b3-4947-8f97-54401e15394a.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @marlonjames71 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by marlonjames71** [reply]
Thanks for moving that for me!

**Comment by jsoref** [comment]
Interesting. This is what I see on GitHub Mobile beta for Android:

![Screenshot_20230309-024235.png](https://user-images.githubusercontent.com/2119212/223953982-8d706c61-e279-45ec-b308-afd1a63673ef.png)

![Screenshot_20230309-024223.png](https://user-images.githubusercontent.com/2119212/223954001-8ac6834e-a0a4-4404-8a57-50a37bb5e438.png)

The second behavior is obviously better than the behavior you're seeing on iOS.

I suspect you could work around this for an individual repository by compacting the images.

**Reply by marlonjames71** [reply]
Oh that is interesting. This has never worked for me, so I don't know what's going on.
